docs/product-installation-guide/src/main/asciidoc/clustering-ps-configure-domain-proc.adoc

```diff
@@ -32,6 +32,37 @@ The `datasource-jndi-name` is the JNDI name of the database specified at the beg
 </data-stores> 
 </timer-service>
 ----
++
+The following table lists the supported databases and the corresponsing `database` attribute value:
++
+[cols="1,1", frame="all", options="header"]
+|===
+|Database
+|Attribute value
+
+
+|Hyper SQL
+|`hsql`
+
+|PostgreSQL
+|`postgresql`
+
+|Oracle
+|`oracle`
+
+
+|IBM DB2
+|`db2`
+
+
+|Hyper SQL
+|`mssql`
+
+|MySQL and MariaDB
+|`mssql`
+
```

Comment:
Please add there also Sybase - https://github.com/wildfly/wildfly/blob/master/feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/ejb3/main/timers/timer-sql.properties#L13

Reply:
True that Sybase is supported for EJB timers. But it's not a supported database for RHPAM currently, so I wouldn't add it right now.

Reply:
I confirmed with Lukas that Sybase isn't currently supported for RHPAM.